app/controllers/docker_manager/admin_controller.rb

```diff
@@ -174,5 +179,12 @@ def repo_version(repo)
     def concat_repos_versions(repos)
       repos.map(&:latest_local_commit).join(", ")
     end
+
+    def discourse_update_pending?(min_discourse_commit)
+      discourse_repo = DockerManager::GitRepo.discourse_repo
+      discourse_repo.update_remote!
+
+      discourse_repo.commits_behind(target: min_discourse_commit) > 0
```

Comment:
If we go for this appoach, we need to make sure it works with the shallow-cloned version of Discourse we include in the base image

Reply:
I just updated the PR to use a different approach.